vignettes/rd-formatting.Rmd

```diff
@@ -263,6 +265,73 @@ Markdown syntax for inline images works. The image files must be in the `man/fig
 #' ![](example-plot.jpg "Example Plot Title")
 ```
 
+# Dynamic R code
+
+Similarly to the knitr package, you can use the markdown inline code markup to evaluate R code and insert its output into the manual page.
+To do this, prefix the code with `r `, i.e. the lowercase letter 'r' and a space character.
+Roxygen will interpret the rest of the text within backticks as R code and evaluate it, and replace the backtick expression with its value.
```

Comment:
, which will be interpreted as markdown.

Reply:
That would be a bit misleading, because the inserted value is not parsed as markdown, so I added this:
> After all such substitutaions, the text of the whole tag is interpreted as markdown, as usual.

OK?

Reply:
:+1: